src/main/java/code401challenges/QuickSort.java

```diff
@@ -1,5 +1,5 @@
 package code401challenges;
-
+//help From geeks for geeks
 public class QuickSort {
     public static int[] quickSort(int[]arr, int left,int right) {

```

Comment:
You could also write you public method signature like: `public static int[] quickSort(int[]arr)` so that users don't have to specify what part of the array they want to sort.  Then have a helper method that takes in the left and right.